javascript/javascript1/week3/smartPhoneUsageApp.js

```diff
@@ -0,0 +1,42 @@
+//Smart-Phone Usage app:
+//Adding an activity
+let activities = [];
+const limit = 110;
+
+function addActivity(date, activity, duration)
+{
+  if(!date){
+    let todayDate = new Date();
+    date = todayDate.toLocaleDateString();
+  }
```

Comment:
Nice! 
`todayDate` be a `const` though.

You could also make use of "default parameters":
```javascript
function addActivity(
  date = new Date().toLocaleDateString, 
  activity, 
  duration
) {
```

This will set `date` to `new Date().toLocaleDateString()` if no date is provided.

Reply:
Done